lib/ddlog_std.rs

```diff
@@ -202,81 +203,88 @@ pub fn bigint_pow32(base: &ddlog_bigint::Int, exp: &u32) -> ddlog_bigint::Int {
 }
 
 // Option
-pub fn option2std<T>(x: ::std::option::Option<T>) -> Option<T> {
+pub fn option2std<T>(x: StdOption<T>) -> Option<T> {
     match x {
-        ::std::option::Option::None => Option::None,
-        ::std::option::Option::Some(v) => Option::Some { x: v },
+        StdOption::None => Option::None,
+        StdOption::Some(v) => Option::Some { x: v },
     }
 }
 
-pub fn std2option<T>(x: Option<T>) -> ::std::option::Option<T> {
+pub fn std2option<T>(x: Option<T>) -> StdOption<T> {
     match x {
-        Option::None => ::std::option::Option::None,
-        Option::Some { x } => ::std::option::Option::Some(x),
+        Option::None => StdOption::None,
+        Option::Some { x } => StdOption::Some(x),
     }
 }
 
-impl<T> From<::std::option::Option<T>> for Option<T> {
-    fn from(x: ::std::option::Option<T>) -> Self {
+impl<T> From<StdOption<T>> for Option<T> {
+    fn from(x: StdOption<T>) -> Self {
         option2std(x)
     }
 }
 
 // this requires Rust 1.41+
-impl<T> From<Option<T>> for ::std::option::Option<T> {
+impl<T> From<Option<T>> for StdOption<T> {
     fn from(x: Option<T>) -> Self {
         std2option(x)
     }
 }
 
-impl<A: FromRecord + serde::de::DeserializeOwned + Default> FromRecord for Option<A> {
-    fn from_record(val: &Record) -> result::Result<Self, String> {
+impl<T> FromRecord for Option<T>
+where
+    T: FromRecord + DeserializeOwned + Default,
+{
+    fn from_record(val: &Record) -> StdResult<Self, String> {
         match val {
             Record::PosStruct(constr, args) => match constr.as_ref() {
                 "ddlog_std::None" if args.len() == 0 => Ok(Option::None {}),
                 "ddlog_std::Some" if args.len() == 1 => Ok(Option::Some {
-                    x: <A>::from_record(&args[0])?,
+                    x: <T>::from_record(&args[0])?,
                 }),
-                c => result::Result::Err(format!(
+                c => StdResult::Err(format!(
                     "unknown constructor {} of type Option in {:?}",
                     c, *val
                 )),
             },
+
             Record::NamedStruct(constr, args) => match constr.as_ref() {
                 "ddlog_std::None" => Ok(Option::None {}),
                 "ddlog_std::Some" => Ok(Option::Some {
-                    x: arg_extract::<A>(args, "x")?,
+                    x: arg_extract::<T>(args, "x")?,
                 }),
-                c => result::Result::Err(format!(
+                c => StdResult::Err(format!(
                     "unknown constructor {} of type Option in {:?}",
                     c, *val
                 )),
             },
-            /* `Option` encoded as an array of size 0 or 1.  This is, for instance, useful when
-             * interfacing with OVSDB. */
+
+            // `Option` encoded as an array of size 0 or 1.  This is, for instance, useful when
```

Comment:
Just curious, why the strong preference for single-line comments?

Reply:
They're more standard in rust I guess, multiline comments aren't used that often